extensions/wikia/ArticleVideo/scripts/ooyala/skin.json

```diff
@@ -0,0 +1,265 @@
+{
+  "general": {
+    "watermark": {"imageResource": {"url": "","androidResource" : "logo","iosResource" : "logo"  },
+                  "position": "bottomRight",
+                  "clickUrl": "",
+                  "target": "_blank",
+                  "transparency": 1,
+                  "scalingOption": "default",
+                  "scalingPercentage": 20
+    },
+    "loadingImage": {"imageResource": {"url": "//player.ooyala.com/static/v4/stable/4.10.6/skin-plugin/assets/images/loader_svg.svg" } },
+    "accentColor":"#448aff"
+  },
+  "localization": {
+    "defaultLanguage": "en",
+    "availableLanguageFile": [
+      {
+        "language": "en",
+        "languageFile": "//player.ooyala.com/static/v4/stable/4.6.9/skin-plugin/en.json",
+        "androidResource": "skin-config/en.json",
+        "iosResource": "en"
+      },
+      {
+        "language": "es",
+        "languageFile": "//player.ooyala.com/static/v4/stable/4.6.9/skin-plugin/es.json",
```

Comment:
We load all the language files on the page load. Can we remove the non-english ones for now? We should figure out how to load only the one of them before we go multilingual.